src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -12,111 +12,104 @@ import * as Utils from 'src/libs/utils'
+      strong(['Pending']), `.`
     ]),
-    h(IdContainer, [id => div([
-      h(FormLabel, { htmlFor: id }, ['Please tell us why']),
-      h(TextArea, {
-        id,
-        style: { height: 100, marginBottom: '0.5rem' },
-        placeholder: 'Enter your reason',
-        value: reason,
-        onChange: setReason
-      }),
-      sendCopyEnabled && div([
-        h(LabeledCheckbox, {
-          checked: sendCopy,
-          onChange: setSendCopy
-        }, [
-          label({ style: { margin: '0 2rem 0 0.25rem' } }, [`Also send me a copy`])
-        ])
-      ])
-    ])]),
-    table({ style: { margin: '1rem', width: '100%' } }, [
-      thead([
-        tr({ style: { height: '2rem' } }, [th({ style: { textAlign: 'left' } }, ['Datasets']), th({ style: { textAlign: 'left', width: '15rem' } }, ['Access'])])
+    div({ style: { marginTop: 30, display: 'flex', flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center' } }, [
+      `Let's continue testing the BETA Data Catalog!`,
+      h(ButtonPrimary, {
+        onClick: onDismiss,
+        style: { padding: '20px 30px' }
+      }, 'OK')
+    ])
+  ]) :
+    h(Modal, {
+      title: 'Request Access',
+      width: '40rem',
+      showButtons: false,
+      showX: true,
+      onDismiss
+    }, [
+      div([
+        'You cannot access this dataset because it is protected by an Authorization Domain. ',
+        'You need to obtain permission from the owner(s) of this dataset in order to get access. ',
+        'Clicking the "Request Access" button below will send an email to the admins of that dataset.'
       ]),
-      tbody(
-        _.map(({ 'dct:title': title, access, 'dct:identifier': id, contacts }) => tr({ key: id, style: { height: '2rem' } }, [
-          td({ style: { paddingRight: 20 } }, [
-            title,
-            div({ style: { fontSize: '.7rem', marginTop: 5, width: 'fit-content' } }, [
-              _.map(({ email }) => email && h(Link, { key: email, href: `mailto:${email}`, style: { marginTop: 5, display: 'block' } }, [email]), contacts)
-            ])
-          ]),
-          td([
-            Utils.switchCase(access,
-              ['Controlled', () => h(RequestDatasetAccessButton, { title, id })],
-              ['Pending', () => span({ style: { fontWeight: 600 } }, ['Request Pending'])],
-              [Utils.DEFAULT, () => span({ style: { fontWeight: 600 } }, ['Permission Granted'])]
-            )
+      h(IdContainer, [id => div([
+        h(FormLabel, { htmlFor: id }, ['Please tell us why']),
+        h(TextArea, {
+          id,
+          style: { height: 100, marginBottom: '0.5rem' },
+          placeholder: 'Enter your reason',
+          value: reason,
+          onChange: setReason
+        }),
+        sendCopyEnabled && div([
+          h(LabeledCheckbox, {
+            checked: sendCopy,
+            onChange: setSendCopy
+          }, [
+            label({ style: { margin: '0 2rem 0 0.25rem' } }, [`Also send me a copy`])
```

Comment:
backticks...not sure if it's a problem...figured there may be a suuuper tiny perf gain

Reply:
i'll change this one, but please let me know what you think about the backticks in the other sections!